src/pybind/mgr/CMakeLists.txt

```diff
@@ -22,4 +22,9 @@ install(DIRECTORY
   REGEX "rook/rook-client-python.*" EXCLUDE
   REGEX "osd_perf_query/.*" EXCLUDE
   REGEX "tox.ini" EXCLUDE
-  REGEX "requirements.txt" EXCLUDE)
+  REGEX "\..+" EXCLUDE
```

Comment:
This seems to match everything on alpine:
https://gitlab.alpinelinux.org/a16bitsysop/aports/-/jobs/395447/raw

Reply:
Argh, perhaps it needs to be double escaped `\\..+`?

Reply:
will try that

Reply:
That is the same:
https://gitlab.alpinelinux.org/a16bitsysop/aports/-/jobs/395565/raw

Without the line it builds like this:
https://gitlab.alpinelinux.org/a16bitsysop/aports/-/jobs/395538/raw

Reply:
I changed to more specific names instead of the one line